data/coalition jobs.txt

```diff
@@ -63,7 +63,7 @@ mission "Coalition: Ringworld Workers 3"
 		"coalition jobs" ++
 		payment
 		payment 20000
-		dialog `You drop off the volunteers on the <planet>. A few minutes later you receive a transfer of <payment>.`
+		dialog `You drop off the volunteers on the <planet>. A few minutes later, you receive a transfer of <payment>.`
```

Comment:
all these ,'s aren't needed though, not sure what grammar rule applies though (I don't do those)